CHANGELOG.md

```diff
@@ -404,8 +404,40 @@ releases.
     `example/cumulus-tf/sync_granule_workflow.asl.json` to include `ACL`
     parameter.
 
+## [v11.1.7] 2022-10-05 [BACKPORT]
+**Please note** changes in 11.1.7 may not yet be released in future versions, as
```

Comment:
I think we need blank lines around this note? My linter throws an error but it may just be warning in CI.